tests/model_registry/rbac/conftest.py

```diff
@@ -2,20 +2,24 @@
 import shlex
 import subprocess
 import os
-from typing import Generator, List, Dict, Any
+import time
+from contextlib import contextmanager
+
+from typing import Callable, ContextManager, Generator, List, Dict, Any
 from ocp_resources.namespace import Namespace
 from ocp_resources.service_account import ServiceAccount
 from ocp_resources.role_binding import RoleBinding
 from ocp_resources.role import Role
 from kubernetes.dynamic import DynamicClient
 from pyhelper_utils.shell import run_command
 from tests.model_registry.utils import generate_random_name, generate_namespace_name
-from simple_logger.logger import get_logger
 from tests.model_registry.constants import MR_INSTANCE_NAME
+from simple_logger.logger import get_logger
 
 
 LOGGER = get_logger(name=__name__)
 DEFAULT_TOKEN_DURATION = "10m"
+SLEEP_TIME = 5
```

Comment:
if you use a constant, then please add unit in the name. Otherwise, it is ok to use 5 directly where you need to.

Reply:
code removed